tests/end-to-end/__init__.py

```diff
@@ -0,0 +1,3 @@
+import pytest
+
+pytestmark = [pytest.mark.end_to_end]
```

Comment:
what does this do? `pytestmark` is only defined never referenced and I can't find any docs for it.

Reply:
This marks all tests in this file with that mark....instead of having to mark each test

Reply:
True dat https://docs.pytest.org/en/latest/example/markers.html#marking-whole-classes-or-modules